src/types.ts

```diff
@@ -39,4 +40,4 @@ export type Code = { jsx: string; css: string };
 
 export type Tab = "jsx" | "css";
 
-export type EditorTheme = "light" | "dark";
+export type EditorTheme = "light" | "dark" | Extension;
```

Comment:
Is it customizable by users?

Reply:
in `LiveEdit` for example I'm passing an extension (`vscode` theme)
it is customizable to "light" | "dark" by users but not allowing (yet, you had a good point) to pass their own theme